docs/pages/web/helpbutton.js

```diff
@@ -0,0 +1,217 @@
+  <Text>Pins created in the last 30 days</Text>
+  <HelpButton
+    text="Date range must include at least 1 day in the last 30 days"
+    accessibilityPopoverLabel="Date range must include at least 1 day in the last 30 days"
+  />
+</Flex>
+`}
+          />
+
+          <Example
+            id="no-final-version-dont"
+            name="How to not use"
+            defaultCode={`
+<Text inline>
+  See the total impact of your paid and organic content working together to increase page
+  visits
+  <HelpButton
+    text="Number of times people visited your website after seeing your content on Pinterest"
+    accessibilityPopoverLabel="Number of times people visited your website after seeing your content on Pinterest"
+  />
+  {' '}add-to-carts{' '}
+  <HelpButton
+    text="Number of times people added your products to their carts after interacting with your Pins"
+    accessibilityPopoverLabel="Number of times people added your products to their carts after interacting with your Pins"
+  />
+  {' '}and checkouts
+  <HelpButton
+    text="Number of checkouts stemming from your Pins and ads"
+    accessibilityPopoverLabel="Number of checkouts stemming from your Pins and ads"
+  />
+  .
+</Text>
+`}
+          />
+        </MainSection.Subsection>
+
+        <MainSection.Subsection columns={2}>
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description="Place HelpButton to the end of the element you wish to add guidance to."
+            sandpackExample={
+              <SandpackExample code={endGuideElement} name="End of guide element" hideEditor />
+            }
+          />
+
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description="Do not include HelpButton within blocks of content."
+            sandpackExample={
+              <SandpackExample
+                code={intermixedBlocks}
+                name="Intermixed guide element blocks"
+                hideEditor
+                hideControls
+              />
+            }
+          />
+        </MainSection.Subsection>
+      </MainSection>
+
+      <MainSection name="Accessibility">
+        <MainSection.Subsection description="Coming soon!" />
+      </MainSection>
+
+      <MainSection name="Localization">
+        <MainSection.Subsection description="Be sure to localize `text`, `accessibilityLabel`, `linkText` and `accessibilityLinkLabel` within HelpButton. Note that localization can lengthen text by 20 to 30 percent." />
+      </MainSection>
+
+      <MainSection name="Variants">
+        <MainSection.Subsection
+          title="With a link"
+          description="Useful for when it's necessary to link to additional information from HelpButton's help text. HelpButton with link interaction can be paired with OnLinkNavigationProvider. See [OnLinkNavigationProvider](https://gestalt.pinterest.systems/web/utilities/onlinknavigationprovider) to learn more about link navigation."
```

Comment:
We only use the full URL in prop description links, as those are also available in the VS Code extension. Everywhere else you only need to include the parts after the domain.